app/src/main/java/com/example/songme/utils/Constants.kt

```diff
@@ -20,4 +20,7 @@ object Constants {
     const val DEFAULT_POSITION = 0
     const val POSITION_VALUE_ONE = 1
     const val MP3_EXTENSION = ".mp3"
+    const val GENRES_KEY = "GENRES_KEY"
+    const val TRACKS_KEY = "TRACKS_KEY"
+    const val IMAGE_KEY = "IMAGE_KEY"
```

Comment:
Xem lại cách đặt tên nhé 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#210-string-constants-naming-and-values
Chỉ cần để private trong companion object của Fragment là được